### PR TITLE
ses_account email_defaults

### PR DESCRIPTION
**Pull request by anmironov**
for ses_account I see in some cases that
        email_defaults:
            from: <email address of service account>
is necessary (as in the example for Microsoft Exchange).

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against main? Unless there is a good reason otherwise, we prefer pull requests against main and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

File: x-pack/docs/en/watcher/actions/email.asciidoc [hunk 1]
```diff
@@ -379,14 +379,18 @@ http://aws.amazon.com/ses[Amazon Simple Email Service] (SES) SMTP service:
...
             auth: true
             starttls.enable: true
             starttls.required: true
-            host: email-smtp.us-east-1.amazonaws.com <1>
+            host: email-smtp.us-east-1.amazonaws.com <2>
             port: 587
             user: <username>
 --------------------------------------------------
+<1> Some organizations configure Amazon SES to validate that the `from` field is a
+    valid local email account.
```
Comment (gmarouli) [inline x-pack/docs/en/watcher/actions/email.asciidoc:393]: What would you think about rephrasing this in a way that is inline with the comment below. For example, (this is just a suggestion):

```
<1> in certain cases `email_defaults.from` is validated by Amazon SES to ensure that it is a valid local email account.
```

**Review by gmarouli — CHANGES_REQUESTED** [review]
Hi @anmironov , thank you for improving the docs. I had 2 small comments, let me know what you think :).

**Comment by abdonpijpelink** [comment]
@gmarouli After chatting with @anmironov , I have applied your suggestions.

**Comment by abdonpijpelink** [comment]
@elasticmachine update branch

**Review by gmarouli — APPROVED** [review]
LGTM :) thank you for helping us out.

**Pull request merged**